### ISSUE TITLE
VOC2011 training ends up with unchanged acc

### ISSUE THREAD

**Issue by white2018** [JihongJu/keras-fcn#33]
hi @JihongJu 

Thanks for sharing fcn program.

I followed your instruction to train voc2011. the intermediate results is shown below. Since three epoch, the loss did not change too much. After ten epoch, the acc even become constant.

I trained with batch_size of 1 or 4, which leads to no change at all. Also, I tried training with or without weights, and still no change.

Epoch 1/100
278/278 [==============================] - 82s 295ms/step - loss: 14.3584 - acc: 0.7594 - val_loss: 3.5362 - val_acc: 0.7539

Epoch 00001: val_loss improved from inf to 3.53623, saving model to ./model/pascal.hdf5
Epoch 2/100
278/278 [==============================] - 80s 287ms/step - loss: 2.5873 - acc: 0.7622 - val_loss: 2.0838 - val_acc: 0.7539

Epoch 00002: val_loss improved from 3.53623 to 2.08382, saving model to ./model/pascal.hdf5
Epoch 3/100
278/278 [==============================] - 80s 288ms/step - loss: 1.8258 - acc: 0.7622 - val_loss: 1.6789 - val_acc: 0.7539

Epoch 00003: val_loss improved from 2.08382 to 1.67890, saving model to ./model/pascal.hdf5
Epoch 4/100
278/278 [==============================] - 80s 288ms/step - loss: 1.5551 - acc: 0.7622 - val_loss: 1.4830 - val_acc: 0.7539

Epoch 00004: val_loss improved from 1.67890 to 1.48300, saving model to ./model/pascal.hdf5
Epoch 5/100
278/278 [==============================] - 80s 287ms/step - loss: 1.4069 - acc: 0.7622 - val_loss: 1.4185 - val_acc: 0.7539
Epoch 00005: val_loss improved from 1.48300 to 1.41854, saving model to ./model/pascal.hdf5
Epoch 6/100
278/278 [==============================] - 80s 287ms/step - loss: 1.3249 - acc: 0.7622 - val_loss: 1.3197 - val_acc: 0.7539

Epoch 00006: val_loss improved from 1.41854 to 1.31969, saving model to ./model/pascal.hdf5
Epoch 7/100
278/278 [==============================] - 80s 287ms/step - loss: 1.2747 - acc: 0.7622 - val_loss: 1.2816 - val_acc: 0.7539

Epoch 00007: val_loss improved from 1.31969 to 1.28161, saving model to ./model/pascal.hdf5
Epoch 8/100
278/278 [==============================] - 79s 285ms/step - loss: 1.2439 - acc: 0.7622 - val_loss: 1.2859 - val_acc: 0.7539

Epoch 00008: val_loss did not improve from 1.28161
Epoch 9/100
278/278 [==============================] - 80s 286ms/step - loss: 1.2223 - acc: 0.7622 - val_loss: 1.2398 - val_acc: 0.7539

Epoch 00009: val_loss improved from 1.28161 to 1.23977, saving model to ./model/pascal.hdf5
Epoch 10/100
278/278 [==============================] - 79s 286ms/step - loss: 1.2084 - acc: 0.7622 - val_loss: 1.2319 - val_acc: 0.7539

Epoch 00010: val_loss improved from 1.23977 to 1.23188, saving model to ./model/pascal.hdf5
Epoch 11/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1997 - acc: 0.7622 - val_loss: 1.2195 - val_acc: 0.7539

Epoch 00011: val_loss improved from 1.23188 to 1.21950, saving model to ./model/pascal.hdf5
Epoch 12/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1930 - acc: 0.7622 - val_loss: 1.2187 - val_acc: 0.7539

Epoch 00012: val_loss improved from 1.21950 to 1.21867, saving model to ./model/pascal.hdf5
Epoch 13/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1903 - acc: 0.7622 - val_loss: 1.2112 - val_acc: 0.7539

Epoch 00013: val_loss improved from 1.21867 to 1.21119, saving model to ./model/pascal.hdf5
Epoch 14/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1894 - acc: 0.7622 - val_loss: 1.2606 - val_acc: 0.7539

Epoch 00014: val_loss did not improve from 1.21119
Epoch 15/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1894 - acc: 0.7622 - val_loss: 1.2147 - val_acc: 0.7539

Epoch 00015: val_loss did not improve from 1.21119
Epoch 16/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1869 - acc: 0.7622 - val_loss: 1.2135 - val_acc: 0.7539

Epoch 00016: val_loss did not improve from 1.21119
Epoch 17/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1831 - acc: 0.7622 - val_loss: 1.2101 - val_acc: 0.7539

Epoch 00017: val_loss improved from 1.21119 to 1.21015, saving model to ./model/pascal.hdf5
Epoch 18/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1850 - acc: 0.7622 - val_loss: 1.2129 - val_acc: 0.7539

Epoch 00018: val_loss did not improve from 1.21015
Epoch 19/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1838 - acc: 0.7622 - val_loss: 1.2188 - val_acc: 0.7539

Epoch 00019: val_loss did not improve from 1.21015
Epoch 20/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1822 - acc: 0.7622 - val_loss: 1.2079 - val_acc: 0.7539

Epoch 00020: val_loss improved from 1.21015 to 1.20793, saving model to ./model/pascal.hdf5
Epoch 21/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1835 - acc: 0.7622 - val_loss: 1.2083 - val_acc: 0.7539

Epoch 00021: val_loss did not improve from 1.20793
Epoch 22/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1817 - acc: 0.7622 - val_loss: 1.2146 - val_acc: 0.7539

Epoch 00022: val_loss did not improve from 1.20793
Epoch 23/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1836 - acc: 0.7622 - val_loss: 1.2028 - val_acc: 0.7539

Epoch 00023: val_loss improved from 1.20793 to 1.20280, saving model to ./model/pascal.hdf5
Epoch 24/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1805 - acc: 0.7622 - val_loss: 1.2089 - val_acc: 0.7539

Epoch 00024: val_loss did not improve from 1.20280
Epoch 25/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1816 - acc: 0.7622 - val_loss: 1.2148 - val_acc: 0.7539

Epoch 00025: val_loss did not improve from 1.20280
Epoch 26/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1830 - acc: 0.7622 - val_loss: 1.2038 - val_acc: 0.7539

Epoch 00026: val_loss did not improve from 1.20280
Epoch 27/100
278/278 [==============================] - 79s 286ms/step - loss: 1.1792 - acc: 0.7622 - val_loss: 1.2053 - val_acc: 0.7539

Epoch 00027: val_loss did not improve from 1.20280
Epoch 28/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1846 - acc: 0.7622 - val_loss: 1.2032 - val_acc: 0.7539

Epoch 00028: val_loss did not improve from 1.20280
Epoch 29/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1808 - acc: 0.7622 - val_loss: 1.2062 - val_acc: 0.7539

Epoch 00029: val_loss did not improve from 1.20280
Epoch 30/100
278/278 [==============================] - 79s 285ms/step - loss: 1.1801 - acc: 0.7622 - val_loss: 1.2030 - val_acc: 0.7539

Epoch 00030: val_loss did not improve from 1.20280

Would you please tell me what's wrong and what should I do for the next.

Looking forward to your reply.

Thanks a lot

**Comment by pgadosey** [comment]
Hi @white2018 Were you able to solve this? I'm having the same issue
